narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,270 @@
+
+[Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+
+[Background on the current hCoV-19 outbreak](https://nextstrain.org/help/coronavirus/SARS-CoV-2).
+
+[FAQs](https://nextstrain.org/help/coronavirus/FAQ).
+
+[Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+
+## External Resources  
+[WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+
+[CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+
+[The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus).
+
+[Our World in Data](https://ourworldindata.org/coronavirus).
+
+[Ask a Scientist & FAQs](https://covid19.fas.org/l/en).
+
+
+```
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text -->
+# [A note on sampling](https://nextstrain.org/ncov/global/2020-04-10?c=country&r=country&d=map&p=grid&legend=closed)
+We currently have sequences from samples taken in 57 countries across 6 continents. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
+<br><br>
+While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
+<br><br>
+For example, the map shows very few sequences from the global south. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Early spread within Asia](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-01-15&p=full)
+
+Here, we see samples from the first month of the outbreak.
+Each tip of the tree represents a sample from an infected individual.
+Each internal node represents an unsampled case, which was the ancestor of all of the other cases within the descendant clade.
+<br><br>
+Tips are colored according to where they were sampled from; nodes are colored by the inferred (most likely) location of the ancestral case.
+<br><br>
+Comparing the trunk of the tree to the leaves helps us understand not only where infections are occurring, but also where they may have been acquired.  
+<br><br>
+During this early period, the majority of sampled and inferred ancestral cases were located in Asia.
+This is consistent with circulation largely within Asia for the first month of the outbreak.  
+
```

Comment:
"The common ancestor of circulating viruses appears to have emerged in Wuhan, China, in late Nov or early Dec 2019."